index.css

```diff
@@ -1,6 +1,100 @@
-header{
```

Comment:
header selector is not closed, please make sure to all your selectors in your CSS